jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java

```diff
@@ -73,7 +75,9 @@ public void send(Process process, List<Span> spans) throws SenderException {
       try {
         responseBody = response.body() != null ? response.body().string() : "null";
       } catch (IOException e) {
-        responseBody = "unable to read response";
+          responseBody = String.format("Unable to read response: %s", e.getMessage());
```

Comment:
I should have caught that on the previous PR, but the indentation here seems odd. Compare with the position of `responseBody` in the `try` part of this try-catch.

Other than that, LGTM.

Reply:
I am using 4 tab indentation. I will change it to 2 tab